crates/ln-dlc-node/src/ln/coordinator_event_handler.rs

```diff
@@ -396,6 +399,12 @@ where
         return Ok(());
     }
 
+    let max_app_fund_amount_msat = max_app_fund_amount * 1000;
+    ensure!(
+        expected_outbound_amount_msat <= (max_app_fund_amount * 1000),
+        "Failed to open channel because maximum fund amount exceeded, {expected_outbound_amount_msat} > {max_app_fund_amount_msat}"
+    );
```

Comment:
I am confused. This is exactly what we are doing now 😅 what do you refer to?

Reply:
I suppose it's pretty close. When I left this comment I didn't yet realise that you had now diverted regular payments away from the `HTLCIntercepted` event too.

I think it would still be cleaner to branch at the very top, before dealing with anything related to the JIT channel case, but we can leave it as is since the payment via `HTLCIntercepted` is no longer supported.